iOS/Flows/Conversation/ConversationListCollectionViewDataSource.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ConversationViewControllerDataSource.swift
+//  ConversationViewControllerDataSource
+//
+//  Created by Martin Young on 9/15/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+typealias ConversationListSection = ConversationListCollectionViewDataSource.SectionType
+typealias ConversationListItem = ConversationListCollectionViewDataSource.ItemType
+
+class ConversationListCollectionViewDataSource: CollectionViewDataSource<ConversationListSection,
+                                                ConversationListItem> {
```

Comment:
Previously called the ConversationCollectionViewDataSource. It's simplified to just handle a conversation LIST collection. The section and item types are a simpler and it only creates one kind of message sequence cell.